demosplan/DemosPlanCoreBundle/Logic/Procedure/ProcedureService.php

```diff
@@ -828,14 +842,36 @@ public function addProcedureEntity(array $data, string $currentUserId): Procedur
                 $data['settings']['emailTitle'] =
                     $this->translator->trans('participation.invitation').': '.($data['name'] ?? '');
             }
-
+            // T34551 all procedures shall get a customer relation
+            // - defalt-customer-blueprint relations are set within the customer only
+            // if a customer is given inside the procedure related $data array then
+            // that signals the procedure should be used as the default-customer-blueprint.
+            $setProcedureAsDefaultCustomerBlueprint = false;
+            if (array_key_exists('custoemr', $data) && $data['customer'] instanceof Customer) {
```

Comment:
typo in custoemr. Did you test this case?

Reply:
This method gets only called when creating a new bluePrint.
As a 'Plattform-Administration' there is a checkbox to mark the blueprint as default.
Interestingly there is never the key 'customer' present and there is simply no option to detect the default-customer-blueprint checkbox was active when creating a NEW procedure.
To set the blueprint as default - an UPDATE has to be triggered.

Fixed the typo - but it has no effect. 2abd20044a336d28b24666c75d053c014998bd18